lib/compat/wordpress-6.1/script-loader.php

```diff
@@ -36,6 +33,27 @@ static function () use ( $style ) {
 	);
 }
 
+/**
+ * This function takes care of adding inline styles
+ * from a Style-Engine store.
+ *
+ * @param string $store_name The name of the store.
+ */
+function gutenberg_enqueue_style_engine_store( $store_name ) {
```

Comment:
I think I'm starting to come around to leaving the work of enqueuing to the consumer like this (I'm slow 😄) especially in the case of Gutenberg, where there are existing functions that we can repurpose.

Maybe later we could think about writing public utility methods in the style engine to take some of the work of enqueuing away from other consumers.

The `gutenberg_enqueue_block_support_styles` function above is another great example of where we could reuse/recycle, especially given the gymnastics we seem to have to go through to ensure backwards compatibility for such functions.